### PR TITLE
bot-proactive message sample migration

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/bot-proactive-messaging-teamsfx/teamsapp.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# yaml-language-server: $schema=https://developer.microsoft.com/json-schemas/teams-toolkit/teamsapp-yaml/1.0.0/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: 1.0.0
+
+projectId: 8e391fef-aec7-4502-a048-4033ad57210c
```
Comment (ChetanSharma-msft) [inline samples/bot-proactive-messaging-teamsfx/teamsapp.yml:6]: Do we need to hardcode ?

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my reviews

File: samples/bot-proactive-messaging-teamsfx/teamsapp.yml (hunk 1 above)
Comment (HuihuiWu-Microsoft) [reply samples/bot-proactive-messaging-teamsfx/teamsapp.yml:6]: Better remove the field.

File: samples/bot-proactive-messaging-teamsfx/env/.env.dev
Comment (HuihuiWu-Microsoft) [inline samples/bot-proactive-messaging-teamsfx/env/.env.dev:1]: Please also include .env.local file

File: samples/bot-proactive-messaging-teamsfx/teamsapp.yml (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/bot-proactive-messaging-teamsfx/teamsapp.yml]: Removed the field

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**